### ISSUE TITLE
missing dependency in documentation to build on Ubuntu 14.04.1

### ISSUE THREAD

**Issue by mschilli87** [The-Compiler/qutebrowser#1200]
according to the [documentation](https://github.com/The-Compiler/qutebrowser/blob/master/INSTALL.asciidoc), the following line should install all dependencies on Ubuntu:

    # apt-get install python3-pyqt5 python3-pyqt5.qtwebkit python-tox python3-sip

Then, one is asked to run 

    $ tox -e mkvenv

This resulted in the following error for me (on Ubuntu 14.04.1):

> ext/_yaml.c:8:22: fatal error: pyconfig.h: No such file or directory

Installing the `python3-dev` package fixed this:

    # apt-get install python3-dev
    $ tox -e mkvenv

>   mkvenv: commands succeeded
>  congratulations :)

**Comment by The-Compiler** [comment]
Hm, interesting - yaml should install successfully even without `python3-dev`, just without C speedups (which are irrelevant for how qutebrowser uses it). Isn't that what you're seeing?

**Comment by mschilli87** [comment]
I can't say much since I'm not a `python` developer. I used `qutebrowser` on Gentoo Linux via the ebuild before. Now I tried to get it running on another machine running Ubuntu by just following the instructions cited above. I simply googled the error message learning that this hints at missing `python-dev` package. Since I know from Gentoo that I built `qutebrowser` for `python` 3.4, I figured I'd need the `python3-dev` package on Ubuntu. After installing that package, `tox` finished without errors. Maybe another package pulled in as one of the `python3-dev` dependencies would have been sufficient, I didn't test that. But with the list currently given in the documentation I couldn't succeed running `tox`.
